packages/sync-actions/test/product-sync-variants.spec.js

```diff
@@ -270,7 +270,7 @@ describe('Actions', () => {
     const now = {
       id: '123',
       masterVariant: {
-        sku: 'v1', key: 'v2', attributes: [{ name: 'foo', value: 'new value' }],
+        id: 1, sku: 'v1', key: 'v2', attributes: [{ name: 'foo', value: 'new value' }],
```

Comment:
The test below felt like it didn't want to test for the creation of the masterVariant. However if there is no id it should be created.